charts/fybrik/files/taxonomy/data_catalog_response.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Catalog Response",
```

Comment:
Title should reflect that this is for get asset details.

Reply:
fixed. Thanks.